.buildkite/steps/tests.sh

```diff
@@ -10,10 +10,12 @@ go install gotest.tools/gotestsum@v1.8.0
 go install github.com/pact-foundation/pact-go/v2@latest
 pact-go -l DEBUG install
 
-# Install node dependencies for js runner tests
+# Install dependencies for js runner tests
 cd internal/runner/testdata
 yarn install
-cd ../../..
+cd playwright
```

Comment:
not quite following this - what creates the playwright directory here?

Reply:
playwright directory contains playwright sample project, commited to this repo.